### PR TITLE
fix(checkbox): fix horizontal spacing between checkboxes and radio buttons

### PR DESCRIPTION
**Pull request by kara**
r: @jelbourn
This PR fixes the baselines for checkboxes and radio buttons stacked vertically. A more robust and permanent solution is needed to fix baselines for checkboxes and radio buttons on the same line.

### CODE REVIEW

File: src/components/checkbox/checkbox.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
 @import "variables";
 
 /** The width/height of the checkbox element. */
-$md-checkbox-size: 18px !default;
+$md-checkbox-size: 20px !default;
```
Comment (jelbourn) [inline src/components/checkbox/checkbox.scss]: Can we refactor this out into something like `$md-toggle-size` and use it for both checkbox and radio button?

Comment (kara) [inline src/components/checkbox/checkbox.scss]: Yes! Good call.

File: src/components/checkbox/checkbox.scss [hunk 2]
```diff
@@ -250,6 +250,10 @@ $_md-checkbox-indeterminate-checked-easing-function: cubic-bezier(0.14, 0, 0, 1)
   }
 }
 
+.md-checkbox-layout label {
+  line-height: 24px;
```
Comment (jelbourn) [inline src/components/checkbox/checkbox.scss]: I'm not sure if we should be setting line-height. I would think that the checkbox should just use whatever line height is already applied. 

@hansl, I think you will have thoughts on this

Comment (kara) [inline src/components/checkbox/checkbox.scss]: The label looks a bit high without it.  I noticed radio buttons were using line-height, so tried to make it consistent.

File: src/core/style/_variables.scss [hunk 3]
```diff
@@ -15,6 +15,11 @@ $z-index-drawer: 100 !default;
 // Global constants
 $pi: 3.14159265;
 
+// Padding between input toggles and their labels
+$md-toggle-padding: 8px;
+// Width and height of input toggles
+$md-toggle-size: 20px;
```
Comment (jelbourn) [inline src/core/style/_variables.scss]: `!default` ?

**Comment by jelbourn** [comment]
LGTM aside from the `!default` comment

File: src/core/style/_variables.scss (hunk 3 above)
Comment (kara) [inline src/core/style/_variables.scss]: Fixed!

**Pull request merged**